imsv-docs-docusaurus/openapi/endpoints/prerequisites/models/get-spending-prerequisite-response.yaml

```
@@ -40,15 +29,33 @@ properties:
             - aml-review
             - set_expected_spend_amount
             - submit_residential_addresses
+            - simulator_call
+        type:
+          deprecated: true
+          type: string
+          enum:
+            - smart_contract_write
+            - kyc
+          description: |
+            The prerequisite instruction type. This attribute is
+            deprecated. Clients should use `stage` and `actionType` instead.
         params:
+          description: |
+            The parameters for the prerequisite instruction. Specific parameters
+            vary based on the type of prerequisite instruction's action type. The action params are:
```

```suggestion
            vary based on the prerequisite instruction's action type. The action params are:
```
Does this simplification still have the same meaning?